process/txsimulator/wrappedAccountsDB.go

```diff
@@ -11,6 +12,8 @@ import (
 
 // readOnlyAccountsDB is a wrapper over an accounts db which works read-only. write operation are disabled
 type readOnlyAccountsDB struct {
```

Comment:
let's rename this to simulationAccountsDB (+ the file) as this no longer is a readonly instance

Reply:
done.